### PR TITLE
feat: added common utils

### PR DESCRIPTION
**Pull request by mansurishmuhametov**
## PR Checklist

Please check if your PR fulfills the following requirements:

-   [x] The commit message follows our guidelines: https://github.com/ngxs/store/blob/master/CONTRIBUTING.md#commit
-   [x] Tests for the changes have been added (for bug fixes / features)
-   [x] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: packages/common/utils/src/check-some-value-is-false.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import { Any } from '@angular-ru/common/typings';
+
+import { isFalse } from './is-false';
+
+export function checkSomeValueIsFalse(...values: Any[]): boolean {
+    if (values.length === 0) {
```
Comment (Schirbak) [inline packages/common/utils/src/check-some-value-is-false.ts:6]: no need to dis

File: packages/common/utils/src/check-every-value-is-true.ts [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import { Any } from '@angular-ru/common/typings';
+
+import { isTrue } from './is-true';
+
+export function checkEveryValueIsTrue(...values: Any[]): boolean {
+    if (values.length === 0) {
```
Comment (Schirbak) [inline packages/common/utils/src/check-every-value-is-true.ts:6]: dis

File: packages/common/utils/src/check-every-value-is-false.ts [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import { Any } from '@angular-ru/common/typings';
+
+import { isFalse } from './is-false';
+
+export function checkEveryValueIsFalse(...values: Any[]): boolean {
+    if (values.length === 0) {
```
Comment (Schirbak) [inline packages/common/utils/src/check-every-value-is-false.ts:6]: and dis

File: packages/common/utils/src/check-some-value-is-true.ts [hunk 4]
```diff
@@ -0,0 +1,7 @@
+import { Any } from '@angular-ru/common/typings';
+
+import { isTrue } from './is-true';
+
+export function checkSomeValueIsTrue(...values: Any[]): boolean {
+    return values.some(isTrue);
```
Comment (Schirbak) [inline packages/common/utils/src/check-some-value-is-true.ts:6]: like this

**Review by Schirbak — CHANGES_REQUESTED** [review]

File: packages/common/utils/src/check-some-value-is-false.ts (hunk 1 above)
Comment (mansurishmuhametov) [reply packages/common/utils/src/check-some-value-is-false.ts:6]: removed
turned out to be superfluous

**Review by Schirbak — DISMISSED** [review]

File: packages/common/utils/src/check-every-value-is-true.ts (hunk 2 above)
Comment (splincode) [inline packages/common/utils/src/check-every-value-is-true.ts:6]: use ternary operator

File: packages/common/utils/src/check-every-value-is-false.ts (hunk 3 above)
Comment (splincode) [inline packages/common/utils/src/check-every-value-is-false.ts:6]: use ternary operator

**Review by splincode — CHANGES_REQUESTED** [review]

File: packages/common/utils/src/check-every-value-is-false.ts (hunk 3 above)
Comment (splincode) [reply packages/common/utils/src/check-every-value-is-false.ts:6]: and this?)

**Review by splincode — APPROVED** [review]

**Review by Schirbak — APPROVED** [review]

**Pull request merged**